### PR TITLE
Create abnormalCoordination.yaml

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
Nico - this pattern will probably only ever be used with Uberon multicellular organism. I can't think of another anatomical entity that one would describe as uncoordinated. :)

### CODE REVIEW

File: src/patterns/dosdp-dev/abnormalCoordinationOfAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+pattern_name: abnormalCoordinationOfAnatomicalEntity
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalCoordinationOfAnatomicalEntity.yaml
+description: "An anatomical entity that has abnormal coordination HP_0011443 Abnormality of coordination."
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormalCoordinationOfAnatomicalEntity.yaml]: @nicolevasilevsky Can we try to explain more what coordination is? And maybe give a more concrete example. :) Thanks!

Comment (nicolevasilevsky) [reply src/patterns/dosdp-dev/abnormalCoordinationOfAnatomicalEntity.yaml]: This is the only HPO term that uses this PATO quality. (There is nothing in MP)

**Comment by nicolevasilevsky** [comment]
dependent on #619

**Comment by matentzn** [comment]
Thanks @nicolevasilevsky! What is the rationale for having this pattern as opposed to abnormalBehavior? Are you worried that coordination and behavior, despite both being in NBO, should not be conflated? If so, we should probably make it a priority to make sure NBO:coordination and NBO:behaviour are seperate branches. @dosumis

**Comment by dosumis** [comment]
Strongly agree - looks like a behavioural phenotype to me: https://hpo.jax.org/app/browse/term/HP:0011443.  We should use NBO and fix that if there  are issues.

**Comment by dosumis** [comment]
Consider obsoleting PATO term?

**Comment by pnrobinson** [comment]
@dosumis  I think that biologists consider Abnormality of coordination to be behavioral, but MDs would consider it to be neurological.

**Comment by dosumis** [comment]
Sounds like another case where it is hard to map clinical specialisms cleanly onto the biology?

What would be a good way to summarise how MDs see this division?   Maybe behavioural = psychiatric; neurological = problem due to systemic nervous system defects?  Presumably division is not always clean , e.g. see https://en.wikipedia.org/wiki/Behavioral_neurology

**Comment by nicolevasilevsky** [comment]
what are the action items here?

Do we want to use NBO instead of PATO? NBO_0000339 'motor coordination'

Should we obsolete this term from PATO?

**Comment by nicolevasilevsky** [comment]
closing - we can use the abnormal behavior pattern for this (at the moment, we don't have a clear distinction between voluntary and involuntary behavior, so they are all subsumed under behavior).

**Pull request closed without merging**